node/core/provisioner/src/lib.rs

```diff
@@ -324,7 +324,15 @@ fn note_provisionable_data(
 				.with_para_id(backed_candidate.descriptor().para_id);
 			per_relay_parent.backed_candidates.push(backed_candidate)
 		},
-		_ => {},
+		// At this time we are not interested in deterring these forms of misbehavior
+		// or issuing a dispute at the backing stage. A dispute for a candidate triggered
+		// at any point before the candidate has been made available, including the
+		// backing stage, can't be guaranteed to conclude. Also candidates which haven't
+		// been made available don't pose a security risk as they can not be included,
+		// approved, or finalized. Thus we wait to punish misbehavior until a candidate
+		// reaches the approval checking stage.
```

Comment:
The main point here is, we cannot dispute at this stage. What the backer could do is record the fact that the candidate was invalid and dispute it once it gets included. This would add to security a bit, but not too much as backers are known ahead of time and we assume they can be dosed. Anyhow if we decide the complexity is worth it, we can do it at some point: https://github.com/paritytech/polkadot/issues/3232 <- Referencing this ticket here might make sense as well.